src/main/java/de/numcodex/feasibility_gui_backend/config/WebSecurityConfig.java

```diff
@@ -104,7 +105,8 @@ public SecurityFilterChain apiFilterChain(
         .requestMatchers(PATH_API_V2 + PATH_QUERY + PATH_QUERY_ID_MATCHER).hasAnyAuthority(keycloakAdminRole, keycloakAllowedRole)
         .requestMatchers(PATH_API_V2 + PATH_QUERY + PATH_QUERY_ID_MATCHER + PATH_SAVED).hasAuthority(keycloakAllowedRole)
         .requestMatchers(PATH_API_V2 + PATH_QUERY + PATH_QUERY_ID_MATCHER + PATH_RESULT).hasAnyAuthority(keycloakAdminRole, keycloakAllowedRole)
-        .requestMatchers(PATH_API_V2 + PATH_QUERY + PATH_QUERY_ID_MATCHER + PATH_RESULT + PATH_DETAILED).hasAuthority(keycloakAdminRole)
+        .requestMatchers(PATH_API_V2 + PATH_QUERY + PATH_QUERY_ID_MATCHER + PATH_RESULT + PATH_DETAILED).hasAnyAuthority(keycloakAdminRole, keycloakAllowedRole)
+        .requestMatchers(PATH_API_V2 + PATH_QUERY + PATH_QUERY_ID_MATCHER + PATH_RESULT + PATH_SITENAMES).hasAuthority(keycloakAdminRole)
```

Comment:
Is it possible to have the auth roles as annotations of the REST controller methods? I find it a bit dangerous to have both informations so far away.